docs/reference/indices/flush.asciidoc

```diff
@@ -39,7 +40,152 @@ or even on `_all` the indices.
 
 [source,js]
 --------------------------------------------------
-$ curl -XPOST 'http://localhost:9200/kimchy,elasticsearch/_flush'
+POST /kimchy,elasticsearch/_flush
 
-$ curl -XPOST 'http://localhost:9200/_flush'
+POST /_flush
 --------------------------------------------------
+// AUTOSENSE
+
+[[indices-synced-flush]]
+=== Synced Flush
+
+Elasticsearch tracks the indexing activity of each shards. Shards that have not
+received any indexing operations for, by default, 30m are automatically marked as inactive. This presents
```

Comment:
s/for, by default, 30m/for 30 minutes (configurable)/

The commas caused my momentary blindness and I didn't read "30m" as 30 minutes - I read it as 30 million and though _I guess I'll figure out how that makes sense later_ on in the document.